contracts/implementation/PoolCommitter.sol

```diff
@@ -342,7 +354,7 @@ contract PoolCommitter is IPoolCommitter, Initializable {
     /**
      * @notice Add the result of a user's most recent commit to their aggregateBalance
      */
-    function updateAggregateBalance(address user) public override {
+    function updateAggregateBalance(address user) public override checkInvariantsAfterFunction {
```

Comment:
Does this need to be public? Seems like its only used internally.

Reply:
In the contracts it is only used internally but it seems weird to restrict people from updating their balance in isolation, hence it remaining public.